packages/orbit-components/src/Loading/Loading.stories.tsx

```diff
@@ -1,123 +1,152 @@
 import * as React from "react";
+import type { Meta, StoryObj } from "@storybook/react";
 
 import TYPE_OPTIONS from "./consts";
-import Card from "../Card";
-import CardSection from "../Card/CardSection";
-import Illustration from "../Illustration";
-import Button from "../Button";
 import RenderInRtl from "../utils/rtl/RenderInRtl";
+import Text from "../Text";
+import Card, { CardSection } from "../Card";
+import Illustration from "../Illustration";
 
 import Loading from ".";
 
-export default {
+const meta: Meta<typeof Loading> = {
   title: "Loading",
-};
-
-export const Default = () => <Loading />;
+  component: Loading,
 
-Default.story = {
   parameters: {
-    info: "This is the default configuration of this component. Visit Orbit.Kiwi for more detailed guidelines.",
+    info: "Check Orbit.Kiwi for more detailed design guidelines.",
   },
-};
-
-export const ButtonLoading = () => <Button loading>Default button</Button>;
 
-ButtonLoading.story = {
-  name: "Button loading",
+  args: {
+    loading: true,
+    text: "Please wait, content is loading...",
+    customSize: 50,
+    type: TYPE_OPTIONS.PAGE_LOADER,
+    id: "loader-id",
```

Comment:
I am thinking if it makes sense to even set all these args here on meta. We don't really need `id` anywhere except Playground, and in most stories you are setting them to `undefined` anyway. Maybe let's just set the default one from the component, so `loading` and `type`? WDYT?